src/main/kotlin/no/nav/familie/ef/sak/vilkår/regler/vilkår/AlderPåBarnRegel.kt

```diff
@@ -42,4 +76,14 @@ class AlderPåBarnRegel : Vilkårsregel(
                 )
             )
     }
+    fun harFullførtFjerdetrinn(fødselsdato: LocalDate): Boolean {
```

Comment:
Lag gjerne noen tester direkte på denne funksjonen:

Hvis vi sender inn "now" til funksjonen er det lettere å teste riktig outcome for forskjellige tilstander:
```
Født | Now | Resultat
2012 | 2022-07-01 | true
2013 | 2022-07-01 | false
2012 | 2022-06-01 | true
2012 | 2022-05-01 | true
2012 | 2022-04-01 | false
2012 | 2021-11-01 | false

```

Reply:
Fixed